suricata.yaml.in

```diff
@@ -986,6 +986,12 @@ asn1-max-frames: 256
 #     hashsize: 2048
 
 ##############################################################################
+# Containers default settings
+# containers:
+#   urlrange:
+#     memcap: 100mb
+#     timeout: 60
```

Comment:
I don't think there is enough info here to have this in the configuration file as-is.. What is a container from the end-user perspective?

Also, as this is part of HTTP, would it not be better under the applayer/http section?  I know the `container` idea is supposed to be generic, but here's we're specifically configuring file storage for http.

Reply:
Indeed, I do not feel very good about the naming. And I hesitated between making something generic or specific...

I will go for making names more specific...